Inspection/build.gradle

```diff
@@ -32,5 +32,5 @@ repositories {
 
 dependencies {
     compile project(':RobotCore')
-    compile project(':OpenFTC')
+    compile project(':openftc')
```

Comment:
Why did you make references to the module lowercase? This is inconsistent with the other modules in the project.